### ISSUE TITLE
is it possible to run this without any manual interaction

### ISSUE THREAD

**Issue by xunxdd** [lebreRafael/inp-lab-test#1]
if this has to be run through devtools, then this script is not really that useful.

Is it possible to run it with any devtool recording etc? and all interactions will be simulated

**Comment by lebreRafael** [comment]
@xunxdd this doesn't have to be run through devtools. You would run that in the terminal.
The Devtools part is to help on writing the script. Just like you asked, you would start recording, interact with your page and devtools will give you the script.
I think I could create a small video to show how to do that and clarify

**Comment by xd-hearst** [comment]
thanks for the reply. @lebreRafael . Just think the tool will be way more useful if there is no manual action involved. Basically, it will be so much nicer if it can find all interaction-able elements and simulate click, typing etc, then generate a report.

Thanks for posting the tool online

**Comment by lebreRafael** [comment]
@xd-hearst Debugbear has [a free tool](https://www.debugbear.com/inp-debugger) which does what you described. Although it seems to only interact with elements above the fold and probably only runs once, so It could be worth to add something like that to inp-lab-test too.

I will consider doing that, thanks for the feedback

**Comment by xd-hearst** [comment]
yeah, debugbear is helpful, however it is very different from google's crux score. So it would be really nice if there is a more comprehensive and automated tool that can scan a whole page. 

Thank you so much
